kpops/components/streams_bootstrap_v3/producer/producer_app.py

```diff
@@ -0,0 +1,121 @@
+import logging
+from functools import cached_property
+
+from pydantic import Field, computed_field
+from typing_extensions import override
+
+from kpops.components.base_components.kafka_app import (
+    KafkaAppCleaner,
+)
+from kpops.components.common.streams_bootstrap import StreamsBootstrap
+from kpops.components.common.topic import (
+    KafkaTopic,
+    OutputTopicTypes,
+    TopicConfig,
+)
+from kpops.components.streams_bootstrap_v3.app_type import AppType
+from kpops.components.streams_bootstrap_v3.kafka_app import KafkaAppV3
+from kpops.components.streams_bootstrap_v3.producer.model import ProducerAppValues
+from kpops.utils.docstring import describe_attr
+
+log = logging.getLogger("ProducerApp")
+STREAMS_BOOTSTRAP_V3 = "3.0.0"
+
+
+class ProducerAppCleaner(KafkaAppCleaner):
+    values: ProducerAppValues
+
+    @property
+    @override
+    def helm_chart(self) -> str:
+        return (
+            f"{self.repo_config.repository_name}/{AppType.CLEANUP_PRODUCER_APP.value}"
+        )
+
+
+class ProducerAppV3(KafkaAppV3, StreamsBootstrap):
```

Comment:
```suggestion
class ProducerAppV3(KafkaAppV3, StreamsBootstrapV3):
```

Reply:
Also there should be some kind of test to ensure it is fixed I guess

Reply:
I have merged `KafkaApp` with `StreamsBootstrap`, added the new configs and created a `StreamsBootstrapV3`

Reply:
Single inheritance and clean abstraction.

Reply:
But KafkaApp handles the deployment of topics and schemas so it should inherit from it

Reply:
Wouldn't this be enough? 🤔 https://github.com/bakdata/kpops/pull/519/files#diff-1d74f42bef3cb5f575f8e945129fcefa298b3dba3c20036470f1819210866058R85

Reply:
I don't like it. Now you have the same logic in two places. KafkaApp was just fine. There was just no need to have values in it